archetypes/learn/module/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+layout: module
+
+# The date represents the date the course was created. Posts with future dates are visible
+# in development, but excluded from production builds. Use the time and timezone-offset
+# portions of of this value to schedule posts for publishing later.
+date: {{ .Date }}
+
+# Draft posts are visible in development, but excluded from production builds.
+# Set this property to `false` before submitting the module for review.
+draft: true
+
+# The description summarizes the course. It appears on the Learn home and module index pages.
+description: Here is a brief description of what this module's all about.
+
+# The meta_desc property is used for targeting search results or social-media previews.
+meta_desc: Here is a brief description of what this module's all about.
+
+# The order in which the module appears on the home page.
+index: 0
+
+# The meta_image appears in social-media previews and on the Learn Pulumi home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for reference.
+meta_image: meta.png
+
+youll_learn:
+    - Stuff
+    - Things
+    - Whatnot
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - christian-nunciato
```

Comment:
Can't the module infer this from the underlying topic authors?

Reply:
Ah, definitely yeah. I'll remove this.